docs/configuration.md

```diff
@@ -715,6 +715,17 @@ Then value of ipfilteredby:
 `aws`: IP from AWS goes to S3 directly
 `awsregion`: IP from certain AWS regions goes to S3 directly, use together with `awsregion`
 
+### `alicdn`
+
+`alicdn` storage middleware allows the registry to serve layers via a content delivery network provided by Alibaba Cloud. Alicdn requires the OSS storage driver.
+
+| Parameter | Required | Description                                           |
+|-----------|----------|-------------------------------------------------------|
+| `baseurl` | yes      | The `SCHEME://HOST` at which Alicdn is served. |
+| `authtype` | yes   | The URL authentication type for Alicdn, which should be `a`, `b` or `c`.     |
```

Comment:
Could you please align the table? Also auth types seem weird. Can you provide CDN documentation link here that describes these auth types?